src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -61,10 +62,15 @@ protected void beforeTest() {
         dbHelper.deleteAll();
 
         // set up processor
+        BlobContainerClientProvider blobContainerClientProvider = mock(BlobContainerClientProvider.class);
+
+        BlobContainerClient blobContainerClient =
+            StorageClientsHelper.getContainerClient(interceptorManager, CONTAINER);
```

Comment:
maybe `StorageClientsHelper` should be renamed (or new helper created).

Reply:
It has more stuff in it, it's more storage related. Plus, these are integration tests, to be revisited.

Reply:
as long as the method explains that the storage client helper method returns a *container* client it's ok I guess.

Reply:
I didn't touch that class, just using one of its existing methods